src/csv_upload/csvUpload.js

```diff
@@ -0,0 +1,217 @@
+import {Address, Email} from '../parties/Children/index'
+import {Charge, Code, Comment, Link, Reference} from '../children/index'
+import {camelCase} from 'lodash'
+
+/**
+ * !This is an internal function that should not be called by the end user!
+ * 
+ * Convert csv object string into object
+ * @function parseString
+ * @param {string} csv
+ * This function ca be used for all object parameters.
+ */
+
+export function parseString({csv})
+{
```

Comment:
So I think there we can do something like
```javascript
export function parseString({ csv, type }) {
// ... etc
}
```
where `type` is a string that determines what kind of data we are parsing. This will probably usually be transactions, but let's include options for assets, parties and books as well just in case.

So calling the function would look something like
```javascript
const data = {
  csv: // csv string here,
  type: 'transactions' // or 'assets', 'parties', 'books'
}
```